feature-libs/product-configurator/rulebased/components/attribute/types/drop-down/configurator-attribute-drop-down.component.ts

```diff
@@ -33,8 +40,40 @@ export class ConfiguratorAttributeDropDownComponent
   attributeDropDownForm = new UntypedFormControl('');
   group: string;
 
+  protected configRouterExtractorService = inject(
+    ConfiguratorRouterExtractorService
+  );
   protected config = inject(Config);
 
+  protected isInitialRenderingOfDomainValues: boolean = true;
+  protected lastAttrSupplement: Configurator.AttributeSupplement | undefined;
+
+  renderDomainValues$: Observable<boolean> = this.isAsyncPricing
```

Comment:
Could you add missing tests? 
![image](https://github.com/SAP/spartacus/assets/61147963/e2d8359d-50b7-487c-a0af-e1065dc0b368)